etc/build/install-minizinc.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# version as of 16 November 2023
+set -o errexit
+set -o nounset
+
+export BIN_DIR=${BIN_DIR:-${HOME}/.local/bin}
+
+rm -rf tmp-install-minizinc
+mkdir tmp-install-minizinc
+pushd tmp-install-minizinc
+git clone https://github.com/MiniZinc/libminizinc.git
+cd libminizinc
+mkdir build
+cd build
+cmake ..
+cmake --build .
+cp minizinc ${BIN_DIR}/minizinc
+mkdir -p ${BIN_DIR}/share
+cp -r ../share/minizinc ${BIN_DIR}/share/minizinc
```

Comment:
should the binary be copied to BIN_DIR/minizinc?

Reply:
We need to be able to call the binary to use solvers that requires minizinc so, I think is better to have it in the path somewhere

Reply:
I mean currently it is not getting copied to the BIN_DIR, right?